src/backend/preprocessor/cfgGenerator/If.js

```diff
@@ -0,0 +1,40 @@
+define([
+    '../Cfg'
+], function (Cfg) {
+    var cfgGenerator;
+    
+    function If(paramNode, options) {
+	var noopInstr = new Cfg({
+	    type: 'NOOP'
+	});
+
+	var tt = cfgGenerator(paramNode.true_body, options);
```

Comment:
casing is wrong (true_body) unless it's something from parser, then better leave that as is :package: